external-crates/move/crates/move-compiler/src/naming/ast.rs

```diff
@@ -691,6 +743,17 @@ impl Value_ {
     }
 }
 
+pub fn explist(loc: Loc, mut es: Vec<Exp>) -> Exp {
+    match es.len() {
+        0 => {
+            let e_ = Exp_::Unit { trailing: false };
+            sp(loc, e_)
+        }
+        1 => es.pop().unwrap(),
+        _ => sp(loc, Exp_::ExpList(es)),
+    }
+}
+
```

Comment:
I'm pretty sure this is just a rebasing artifact from an old branch, as it doesn't appear to be called in the new code.

Reply:
It was at one point. Missed its removal, thanks!